### PR TITLE
Feature/user/can create account

### PR DESCRIPTION
**Pull request by ssoonsl**
Please review. Added navbar in index page as part of user story: User can create account.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  resources :tweets
```
Comment (gohdaniel15) [inline config/routes.rb]: If you do this, you will generate URI's for all 7 routes. However, your controller only defines 1 of the methods. 

What will happen is users who type in a URI that you haven't specified in your controller yet, will receive a nasty error message. 

I suggest putting an `only: [:index]` here. This way, when you add more functionality, you can add more methods into the array.

Comment (vananrajaindern) [reply config/routes.rb]: Please remove this  "resources :users"

Comment (ssoonsl) [reply config/routes.rb]: Gotcha, will limit tweet to [:index] and add onto it as each route is needed. Thanks! :balloon:

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+# of increasing the boot-up time by auto-requiring all files in the support
+# directory. Alternatively, in the individual `*_spec.rb` files, manually
+# require only the support files necessary.
+#
+# Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
+
+# Checks for pending migration and applies them before tests are run.
+# If you are not using ActiveRecord, you can remove this line.
+ActiveRecord::Migration.maintain_test_schema!
+
```
Comment (gohdaniel15) [inline spec/rails_helper.rb:28]: Looks like you forgot to add some configurations for shoulda matcher here 😅

Comment (ssoonsl) [reply spec/rails_helper.rb:28]: Thank you for spotting the error! RSpec tests pass now in next commit :joy:

**Review by gohdaniel15 — APPROVED** [review]
LGTM 🚀

**Pull request merged**